src/hapi.d.ts

```diff
@@ -1,6 +0,0 @@
-import '@hapi/hapi';
```

Comment:
Please delete (via `git rm`) these three `d.ts` files since they no longer have any other content are now blank.

Reply:
They're already deleted on my machine and git refuses to `rm` them because it can't find them. Are you sure they still exist on your machine?

Reply:
That means you used `rm` and not `git rm` first. So in order to `git rm` you’d have to reset those three files via `git checkout — koa.d.ts`

Reply:
I'll see if that works when I'm back at my desk.

In the meantime, are you really really really sure they're not deleted? The PR diff tells me the files have been deleted and they're not in [my fork's tree](https://github.com/HoldYourWaffle/tsoa-iso/tree/remove-user-property/src) either

Reply:
yes I am sure that the hapi file has not been deleted. If it were deleted, then Github wouldn't show the diff like it currently does. It would show something like this:
![image](https://user-images.githubusercontent.com/14020024/66064715-f2134800-e513-11e9-80c3-26785fe0b060.png)

Reply:
This is strange. Now it's showing that the file was deleted. Github must have had a UI/UX bug. My apologies.